src/app/clusters/operational-state-server/operational-state-delegate.h

```diff
@@ -213,10 +213,10 @@ class Delegate
 public:
     /**
      * Get the current operational state.
-     * @param op The GenericOperationalState to fill with the current operational state value.
+     * @param op The uint8_t to fill with the current operational state value.
      * @return void.
      */
-    virtual void GetCurrentOperationalState(GenericOperationalState & op) = 0;
+    virtual void GetCurrentOperationalState(uint8_t & op) = 0;
```

Comment:
```suggestion
    virtual uint8_t GetCurrentOperationalState() = 0;
```
And adjust the documentation accordingly.